backend/src/main/java/ai/verta/modeldb/metadata/MetadataDAORdbImpl.java

```diff
@@ -98,6 +99,31 @@ public void addProperty(Session session, IdentificationType id, String key, Stri
     session.saveOrUpdate(new MetadataPropertyMappingEntity(id0, value));
   }
 
+  @Override
+  public boolean updateProperty(IdentificationType id, String key, String value)
```

Comment:
is it used anywhere?

Reply:
No. I am working on this PR so I will remove it.

Reply:
Method removed